src/components/Details/Details.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
```suggestion
import React, { useState, useEffect } from 'react';
```
try to paste space after you open { and before closing }

Reply:
fixed